packages/spor-card-react-native/src/Card.tsx

```diff
@@ -71,6 +71,7 @@ type CardProps = Exclude<RestyleProps, "elevationLevel"> & {
   onPress?: () => void;
   onClose?: () => void;
   selected?: boolean;
+  disabled?: boolean;
```

Comment:
Jeg er veldig fan av å prefikse alle boolean props med `is` – dvs `isDisabled`. Ser man har introdusert `selected` – det hadde vært fint å endre denne også.

Reply:
Jeg er enig! fikser det! 😊